tests/chainerx_tests/conftest.py

```diff
@@ -65,18 +69,21 @@ def _get_required_cuda_devices_from_device_name(device_name):
     return int(s[1]) + 1
 
 
+def skip_if_no_cuda_device_available(device_name):
```

Comment:
Let's prefix with `_` to explicitly indicate that this function is only used here.